PROJECT

```diff
@@ -1,3 +1,17 @@
+# Code generated by tool. DO NOT EDIT.
+# This file is used to track the info used to scaffold your project
+# and allow the plugins properly work.
+# More info: https://book.kubebuilder.io/reference/project-config.html
 domain: kcl-lang.io
 repo: github.com/kcl-lang/kcl-controller
-version: "1"
+resources:
+- api:
+    crdVersion: v1
+    namespaced: true
+  controller: true
+  domain: kcl-lang.io
+  group: krm.kcl.dev.fluxcd
+  kind: KCLRun
+  path: github.com/kcl-lang/kcl-controller/api/v1alpha1
```

Comment:
s/kcl-controller/flux-kcl-controller

Modifications need to be completed throughout the entire go warehouse.